test/AWS.Deploy.CLI.UnitTests/DockerTests.cs

```diff
@@ -69,7 +96,7 @@ private string ResolvePath(string projectName)
             return Path.Combine(testsPath, "..", "testapps", "docker", projectName);
         }
 
-        private void AssertDockerFilesAreEqual(string path, string generatedFile = "Dockerfile", string referenceFile = "Dockerfile")
+        private void AssertDockerFilesAreEqual(string path, string generatedFile = "Dockerfile", string referenceFile = "ReferenceDockerfile")
```

Comment:
I noticed this test was never doing what it seemed designed to do - compare a newly generated Dockerfile against a static reference file. We have been comparing the generated file against itself.

Updated the test, which also required updating a lot of the reference files to match the current template.